tfunc/tfunc_test.go

```diff
@@ -33,14 +33,24 @@ func TestAllForDups(t *testing.T) {
 
 // Wrap the new template to use our template library
 func NewTemplate(ti hcat.TemplateInput) *hcat.Template {
```

Comment:
Public function comment should start with function name

Reply:
Made this method private as it should have been being only a test utility function.